CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 - Added `DiskIo` and `DiskIo2` protocols.
 - Added `HardDriveMediaDevicePath` and related types.
 - Added `PartialOrd` and `Ord` to the traits derived by `Guid`.
+- Added `TryFrom&lt;core::ffi::CStr&gt;` implementation for `CStr8`.
```

Comment:
`TryFrom&lt;core::ffi::CStr&gt;` -> `TryFrom<core::ffi::CStr>` (HTML escapes in backticks get rendered literally)